Makefile

```diff
@@ -59,7 +59,7 @@ LOCAL_BUILD_TAGS = $(BTRFS_BUILD_TAG) $(LIBDM_BUILD_TAG) $(OSTREE_BUILD_TAG) $(D
 BUILDTAGS += $(LOCAL_BUILD_TAGS)
 
 ifeq ($(DISABLE_CGO), 1)
-	override BUILDTAGS = containers_image_ostree_stub exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
+	override BUILDTAGS = netgo osusergo containers_image_ostree_stub exclude_graphdriver_devicemapper exclude_graphdriver_btrfs containers_image_openpgp
```

Comment:
Thanks for the PR.

- `netgo` bypasses NSS, including the glibc resolver (per https://golang.org/pkg/net/ , e.g. breaking `.local` names.
- `osusergo` bypasses NSS as well, breaking e.g. LDAP uses.

Yes, I realize that these downsides are fundamentally necessary to make static builds work at all, but they are _significant_ downsides.

I’m sorely tempted to just remove any mention of statically-linked or non-CGo builds from the repository (@rhatdan WDYT?).

If not that, at the very least `README` should document these downsides of a static build as well.

Reply:
I would hate to remove them, since it seems many in the GO world seem to want/require them.  I guess Documenting the shortcomings is the best solution.

The NSS stuff is only important for rootless container/storage I believe?